tests/test_report.py

```diff
@@ -11,7 +11,7 @@ def test_shipment_report():
     )
 
     assert report.object == "ShipmentReport"
-    assert report.status == "available"
+    assert report.status in ("available", "new")
```

Comment:
Do you think this is sufficient, or will the retrieve() call fail for new reports? I can't reproduce new reports.